src/microsim/transportables/MSPModel_JuPedSim.h

```diff
@@ -80,26 +80,33 @@ class MSPModel_JuPedSim : public MSPModel {
         Position getPosition(const MSStageMoving& stage, SUMOTime now) const;
         void setPosition(double x, double y);
 
+        Position getPreviousPosition() const;
```

Comment:
This function gets never called

Reply:
Yes. I removed it, there is no need to expose this attribute.